pages/settings.jsx

```diff
@@ -1,4 +1,4 @@
-import { useContext } from 'react';
+import React, { useContext } from 'react';
```

Comment:
Don't need react in the pages. Next does that for us

Reply:
Or the components for that matter

Reply:
ahh, I didnt see this 

```
"rules": {
    "react/react-in-jsx-scope": "off"
  }
```